templates/view.erb

```diff
@@ -34,6 +34,9 @@ view "<%= @name %>" {
 	};
 <%-   end -%>
 <%- end -%>
+<%- if @empty_zones != '' -%>
+	empty-zones-enable <%= @empty_zones ? 'yes' : 'no' %>;
```

Comment:
I think it would be acceptable if the `empty-zones-enable` were always present in the configuration text, and just translated the a boolean-valued parameter into a 'yes' or 'no' string.